install-self-hosted-toolbox

```diff
@@ -75,7 +75,8 @@ fi
 
 
 echo "Installing the when CLI"
-install_cmd mv ~/.toolbox/when $INSTALL_PATH/when
+ERLANG_VERSION=$(erl -eval 'erlang:display(erlang:system_info(otp_release)), halt().'  -noshell | tr -d '"')
```

Comment:
What if Erlang isn't installed?

Reply:
The when CLI installation will fail, but the rest of the toolbox should be installed successfully, which should also be how it behaves currently.
I created a [sample install script](https://github.com/renderedtext/s2-playground/blob/test-when-install-process/test-script.sh) with the when CLI part only and ran it [here](https://rtx.semaphoreci.com/jobs/52c2d811-15a5-41ea-9d08-dda249a256ce#L128) on a MacOS machine that does not have Erlang and the result was as expected.